pkg/apk/apk/implementation.go

```diff
@@ -880,6 +882,11 @@ func (a *APK) fetchAlpineKeys(ctx context.Context, alpineVersions ...string) err
 
 	u := alpineReleasesURL
 	client := a.client
+	// NB: Does not seem to work, but at least offline builds are
```

Comment:
I think we might want to just deprecate and remove `fetchAlpineKeys` and require apko users pulling Alpine packages to explicitly specify the keys they want to use to verify Alpine packages.

This change LGTM but I wouldn't want to keep investing in this code path without corresponding tests, and a commitment that we'll work to keep this working (and I don't think we have that)

Reply:
I agree, but there are current workflow that is broken for a user. This is a minimal change to unbreak this existing feature.

Then we can look into ripping it out. I always am risk averse to rip out broken things. I somehow prefer to fix things up, prove the work, then rip them out.